src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -853,8 +905,31 @@ int fx_muxer_t::read_config_and_execute(
         append_realpath(path, &probe_realpaths);
     }
 
+    // 'No candidate fx roll forward' is disabled by default. It can be enabled through:
+    // 1. Command line argument (--no-candidate-fx-roll-forward)
+    // 2. Runtimeconfig json file ('noCandidateFxRollForward' property)
+    // 3. DOTNET_NO_CANDIDATE_FX_ROLL_FORWARD env var
+    // The conflicts will be resolved by following the priority rank described above (from 1 to 3)
+    bool no_candidate_fx_roll_fwd_enabled = config.get_no_candidate_fx_roll_fwd();
+    if (!no_candidate_fx_roll_fwd.empty())
+    {
+        if (no_candidate_fx_roll_fwd == _X("true"))
+        {
+            no_candidate_fx_roll_fwd_enabled = true;
```

Comment:
I think we should maintain this state as integer (0 or 1), instead of bool, incase we wish to support different policy levels in future for this roll forward. Bool will restrict us from going that route.